From 0d677475b710b9bb61d4b3ac5435c36b47d3a155 Mon Sep 17 00:00:00 2001
From: Peter Simons <simons@cryp.to>
Date: Wed, 8 Feb 2017 11:28:42 +0100
Subject: [PATCH] bash-completion: quote pattern argument to grep

Without the quotes, bash might expand that pattern based on the contents of the
current working directory or -- if nullglob is set -- the argument disappears
outright if no directory entry matches.
---
 scripts/bash/task.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/bash/task.sh b/scripts/bash/task.sh
index e0c7fb03..d15ed3eb 100644
--- a/scripts/bash/task.sh
+++ b/scripts/bash/task.sh
@@ -72,7 +72,7 @@ _task_offer_contexts() {
     COMPREPLY=( $(compgen -W "$($taskcommand _context) define delete list none show" -- $cur) )
 }
 
-_task_context_alias=$($taskcommand show | grep alias.*context | cut -d' ' -f1 | cut -d. -f2)
+_task_context_alias=$($taskcommand show | grep "alias.*context" | cut -d' ' -f1 | cut -d. -f2)
 
 _task()
 {
-- 
2.11.1

